packages/twenty-front/src/modules/ui/input/components/Select.tsx

```diff
@@ -152,10 +152,7 @@ export const Select = <Value extends SelectValue>({
                 <DropdownMenuSeparator />
               )}
               {!!filteredOptions.length && (
-                <DropdownMenuItemsContainer
-                  hasMaxHeight
-                  width={dropDownMenuWidth}
-                >
+                <DropdownMenuItemsContainer hasMaxHeight width={'auto'}>
```

Comment:
For most dropdown, do not set the dropdown width but set the content width
For widthAutoGrow dropdown, do not set the content width (use 'auto')

Reply:
I think that this should become the default strategy, using auto everywhere, which is the default value for width anyway, let's see what generic solution we can find to avoid having constant regressions on this topic.